README.md

```diff
@@ -91,7 +116,16 @@ Define a function called `doSomething`. It should take one argument, called
 
 Your Answer:
 ```js
-// write code here
+
+function thingToDo() {
+  console.log("Things are happening");
+}
+
```

Comment:
This question is also kind of tricky but I think it was looking for you to pass in an argument into doSomething. 

The solution answer is :

```
function doSomething(thingToDo){
  thingToDo();
}
doSomething(function(){
  alert("Yay!");
});
```